Makefile

```
@@ -12,4 +13,17 @@ lint:
 					RESULT=1; \
 				fi; \
 		done; \
+		for d in $$(find . -name '*-mixin' -a -type d -print); do \
+			if [ -e "$$d/jsonnetfile.json" ]; then \
+				echo "Installing dependancies for $$d"; \
```

```suggestion
				echo "Installing dependencies for $$d"; \
```

In case you care about this ;)